src/ripple/app/misc/LoadFeeTrack.h

```diff
@@ -91,49 +85,67 @@ class LoadFeeTrack
 
     std::uint32_t getLoadFactor () const
     {
-        ScopedLockType sl (mLock);
-        return std::max({ mClusterTxnLoadFee, mLocalTxnLoadFee, mRemoteTxnLoadFee });
+        std::lock_guard <std::mutex> sl (lock_);
+        return std::max({ clusterTxnLoadFee_, localTxnLoadFee_, remoteTxnLoadFee_ });
+    }
+
+    std::pair<std::uint32_t, std::uint32_t>
+    getScalingFactors() const
+    {
+        std::lock_guard<std::mutex> sl(lock_);
+
+        return std::make_pair(
+            std::max(localTxnLoadFee_, remoteTxnLoadFee_),
+            std::max(remoteTxnLoadFee_, clusterTxnLoadFee_));
     }
 
 
     void setClusterFee (std::uint32_t fee)
     {
-        ScopedLockType sl (mLock);
-        mClusterTxnLoadFee = fee;
+        std::lock_guard <std::mutex> sl (lock_);
+        clusterTxnLoadFee_ = fee;
     }
 
     bool raiseLocalFee ();
     bool lowerLocalFee ();
 
     bool isLoadedLocal () const
     {
-        ScopedLockType sl (mLock);
-        return (raiseCount != 0) || (mLocalTxnLoadFee != lftNormalFee);
+        std::lock_guard <std::mutex> sl (lock_);
+        return (raiseCount_ != 0) || (localTxnLoadFee_ != lftNormalFee);
     }
 
     bool isLoadedCluster () const
     {
-        ScopedLockType sl (mLock);
-        return (raiseCount != 0) || (mLocalTxnLoadFee != lftNormalFee) || (mClusterTxnLoadFee != lftNormalFee);
+        std::lock_guard <std::mutex> sl (lock_);
+        return (raiseCount_ != 0) || (localTxnLoadFee_ != lftNormalFee) ||
+            (clusterTxnLoadFee_ != lftNormalFee);
     }
 
 private:
-    static const int lftNormalFee = 256;        // 256 is the minimum/normal load factor
-    static const int lftFeeIncFraction = 4;     // increase fee by 1/4
-    static const int lftFeeDecFraction = 4;     // decrease fee by 1/4
-    static const int lftFeeMax = lftNormalFee * 1000000;
-
-    beast::Journal m_journal;
-    using LockType = std::mutex;
-    using ScopedLockType = std::lock_guard <LockType>;
-    LockType mutable mLock;
-
-    std::uint32_t mLocalTxnLoadFee;        // Scale factor, lftNormalFee = normal fee
-    std::uint32_t mRemoteTxnLoadFee;       // Scale factor, lftNormalFee = normal fee
-    std::uint32_t mClusterTxnLoadFee;      // Scale factor, lftNormalFee = normal fee
-    int raiseCount;
+    static constexpr std::uint32_t lftNormalFee = 256;        // 256 is the minimum/normal load factor
```

Comment:
Searching for "static const" vs "static [^ ]* const" (with regexes) finds a nice inconsistent mix (#sarcasm) of both styles, but a huge chunk of the former are in external libraries. I'll change these instances around, but am not going to make any attempt to do a general conversion at this time.